elixir/apps/domain/lib/domain/crypto.ex

```diff
@@ -5,22 +5,34 @@ defmodule Domain.Crypto do
     rand_base64(@wg_psk_length)
   end
 
-  def rand_string(length \\ 16) do
+  def rand_number(length \\ 8) when length > 0 do
```

Comment:
Would it make sense to name this function something like `rand_number_str` since it's return a string?